.github/workflows/bench.yml

```diff
@@ -108,6 +108,7 @@ jobs:
       - name: Run cargo bench
         env:
           NSS_DB_PATH: ${{ github.workspace }}/neqo/test-fixture/db
+          RUST_BACKTRACE: "full"
```

Comment:
Should we always use this in CI for all workflows?

Reply:
@larseggert I will add in another pull request.